src/Raven.Studio/typescript/commands/database/index/clearIndexErrorsCommand.ts

```diff
@@ -0,0 +1,19 @@
+import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+import endpoints = require("endpoints");
+
+class clearIndexErrorsCommand extends commandBase {
+    constructor(private indexesNames: string[], private db: database) {
+        super();
+    }
+
+    execute(): JQueryPromise<void> {
+        const args =  this.indexesNames ? { name: this.indexesNames } : "";
```

Comment:
const args =  this.indexesNames ? { name: this.indexesNames } : {}.

Reply:
done